app/api/chat/route.ts

```
@@ -17,11 +18,28 @@ export async function POST(request: NextRequest) {
+        error:
+          'API key is required. Please provide it in the x-meeting-baas-api-key header.',
+      },
+      { status: 401 },
+    );
+  }
+
+  // Set the API key in the environment for the MCP client
+  process.env.BAAS_API_KEY = apiKey;
+
   try {
     let client = await createMCPClient({
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'client' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)